tests/unit/test_tools_archive.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""Tests for webcompat.com archive tools.
+
+TODO: test that an issue can be on a different repo
```

Comment:
I wonder what this TODO means... 🤔

Reply:
I think it's probably because I was wondering if I would have a dependency inside this on hardcoding the repo path. As I was planning to initiliaze the issue from the URL but I changed my mind, and decided to initialize the issue from a payload. So the fetching of the issue itself is outside of the model and already versatile.